components/password_manager_strings.grdp

```diff
@@ -204,13 +204,37 @@
     <message name="IDS_PASSWORD_MANAGER_BIOMETRIC_AUTHENTICATION_CONFIRMATION_DESCRIPTION_MAC" desc="Text that appears in the description of biometric authentication confirmation popup.">
       You can change whether you use your screen lock for filling passwords in <ph name="SETTINGS">$1<ex>Settings</ex></ph>
     </message>
+    <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_TITLE_BRANDED" desc="Text that appears in the title of the relaunch Brave bubble.">
+      Brave Password Manager needs access to MacOS Keychain
```

Comment:
"Brave Password Manager" sounds like a product, but probably ok?

Reply:
This is an upstream string too, we just replaced `Google` with `Brave`.